packages/kit/src/exports/vite/index.js

```diff
@@ -519,7 +517,7 @@ async function kit({ svelte_config }) {
 				if (vite_config.build.ssr) return;
 
 				const guard = module_guard(this, {
-					cwd: vite.normalizePath(process.cwd()),
+					cwd: vite.normalizePath(cwd),
```

Comment:
should we be introducing a new option for this or using the one that already exists?
```suggestion
					cwd: vite.normalizePath(vite_config.root),
```